analyzers/tests/SonarAnalyzer.UnitTest/Rules/CognitiveComplexityTest.cs

```diff
@@ -33,29 +33,37 @@ public class CognitiveComplexityTest
 
         [TestMethod]
         public void CognitiveComplexity_CS() =>
-            builderCS.AddPaths(@"CognitiveComplexity.cs")
+            builderCS.AddPaths("CognitiveComplexity.cs")
                 .WithOptions(ParseOptionsHelper.FromCSharp8)
                 .Verify();
 
         [TestMethod]
         public void CognitiveComplexity_CS_LocalFunctions() =>
-        builderCS.AddPaths(@"CognitiveComplexity.LocalFunctions.cs")
-            .WithOptions(ParseOptionsHelper.FromCSharp8)
-            .Verify();
+            builderCS.AddPaths("CognitiveComplexity.LocalFunctions.cs")
+                .WithOptions(ParseOptionsHelper.FromCSharp8)
+                .Verify();
 
 #if NET
+
         [TestMethod]
         public void CognitiveComplexity_CS_CSharp9() =>
-            builderCS.AddPaths(@"CognitiveComplexity.CSharp9.cs")
+            builderCS.AddPaths("CognitiveComplexity.CSharp9.cs")
                 .WithTopLevelStatements()
                 .Verify();
 
         [TestMethod]
         public void CognitiveComplexity_CS_CSharp10() =>
-            builderCS.AddPaths(@"CognitiveComplexity.CSharp10.cs")
+            builderCS.AddPaths("CognitiveComplexity.CSharp10.cs")
                 .WithLanguageVersion(LanguageVersion.CSharp10)
```

Comment:
Better to use .WithOptions(ParseOptionsHelper.FromCSharp10) instead of WithLanguageVersion. Locally there is no difference but on the CI, FromChsarp10 will also run it for CSHarp11.